pwa/src/components/TreatmentFields.json

```diff
@@ -0,0 +1,132 @@
+{
+    "treatments": [
+        {
+            "id": "multivitamins",
+            "group-title": "Multivitamins",
+            "group": [
+                { "name": "Multivitamins - Childrens", "id": "vitamin-childrens" },
+                { "name": "Multivitamins - Adult", "id": "vitamin-adult" },
+                { "name": "Multivitamins - Prenatal", "id": "vitamin-prenatal" }
```

Comment:
Are some of the treatments throughout this form mutually exclusive, and should we have checkbox logic that supports that?

Reply:
The greencard does not indicate any mutually exclusive treatments. One reason I could see this not being the best idea is if they prescribe a high dosage of a particular treatment and later want to taper it off by giving the same treatment but smaller dosage. If we turned all of these cases into radio buttons (functionally that is what they would be), we would miss out on that. What are your thoughts on this?

Reply:
I'd like to suggest keeping them as not mutually exclusive for now until we can optimize with CHI input.

Reply:
sounds good, consider this resolved.